lib/models/WebsiteConfiguration.ts

```diff
@@ -22,7 +22,7 @@ export type Condition = {
     httpErrorCodeReturnedEquals?: string;
 };
 
-export type RoutingRuleParams = { redirect: Redirect; condition?: Condition };
+export type RoutingRuleParams = { redirect?: Redirect; condition?: Condition };
```

Comment:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/aws-sdk2-types/clients/s3.d.ts#L6279

I don't see the redirect being optional